Automattic/jetpack
# Required Reviews: Allow plugin-touching teams to approve other plugin composer changes

PR attempts to allow anyone in one of the plugin teams to approve a PR that touches the composer.{json|lock} of other plugins.

Closes #23913

#### Changes proposed in this Pull Request:

* Update required appovers for Composer files.

#### Jetpack product discussion


p9dueE-4SE-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Open a PR off of this branch that touches jetpack's composer.json.
* Have someone off of Crew, on a team like Heart of Gold, Backup, Scan, Search, etc attempt to approve it.

See test run at https://github.com/Automattic/jetpack/runs/5997786333?check_suite_focus=true
